tests/helpers/volumes.py

```diff
@@ -232,3 +233,62 @@ def get_volume_and_unit(ansible_zos_module, path):
     list_volumes = [[x[1], x[2]] for x in sorted_volumes]
 
     return list_volumes
+
+def get_volumes_sms_mng_class(ansible_zos_module, volumes_on_system):
```

Comment:
Is `mng` short for management? the most commont abreviation is `mgmt`. Btw the electronic music group MGMT is actually "The management" :) https://www.youtube.com/watch?v=fe4EK4HSPkI
```suggestion
def get_volumes_sms_mgmt_class(ansible_zos_module, volumes_on_system):
```